nodes/src/nodes/batch_processor/IGlobal.py

```
@@ -0,0 +1,85 @@
+            )
+        except Exception as e:
+            warning(str(e))
+            raise
+
+    def validateConfig(self) -> None:
+        try:
+            cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+            concurrency = _int(cfg, 'concurrency', 4, lo=1, hi=64)
+            if concurrency < 1:
+                warning('concurrency must be at least 1')
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)